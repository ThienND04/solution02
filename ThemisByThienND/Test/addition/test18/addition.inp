10
2035 5252 2442 443658
1527 1481 1963 850878
2300 5472 2473 438165
2272 5399 3784 922503
1676 3506 2903 542612
2079 5262 4984 197157
2451 1874 4762 380278
1575 4186 1886 707826
1393 5498 3207 277633
1370 2128 4882 758821
