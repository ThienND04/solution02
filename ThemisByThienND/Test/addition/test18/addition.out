0
83
2
12
9
0
3
18
2
0
